stdlib/public/core/IntegerParsing.swift

```diff
@@ -102,7 +102,7 @@ extension FixedWidthInteger {
   ///
   /// The string passed as `text` may begin with a plus or minus sign character
   /// (`+` or `-`), followed by one or more numeric digits (`0-9`) or letters
-  /// (`a-z` or `A-Z`). The string is case insensitive.
+  /// (`a-z` or `A-Z`). The string is case-insensitive.
```

Comment:
Okay, really the string is just a string. 😅 Can you change this to "The string parsing is case insensitive"?